package/MDAnalysis/coordinates/PDB.py

```diff
@@ -651,7 +700,25 @@ def _write_pdb_header(self):
             self.REMARK(*remarks)
         except AttributeError:
             pass
-        self.CRYST1(self.convert_dimensions_to_unitcell(u.trajectory.ts))
+
+        if u.dimensions is None or np.allclose(u.dimensions, np.zeros(6)):
```

Comment:
Maybe add a comment for ourselves that we internally consider both None and zeros equivalent, and that #2698 will address this.